IFC4x3/PropertyConstants/c/CONCAVE_CONVEX_2ce5d2r5bFARqt407gptBA.xml

```diff
@@ -0,0 +1,3 @@
+<?xml version="1.0" encoding="utf-8"?>
+<DocPropertyConstant xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance" id="CONCAVE_CONVEX_2ce5d2r5bFARqt407gptBA" Name="CONCAVE_CONVEX" UniqueId="a6a059c2-d459-4f29-bd37-1001eacf72ca" />
```

Comment:
Description.

Reply:
Fixed in [26b41fa](https://github.com/bSI-InfraRoom/IFC-Specification/pull/828/commits/26b41fa080bec029809d1e01e57a3b4e2c4d9122)